src/components/calcite-button/resources.ts

```diff
@@ -0,0 +1,7 @@
+export const CSS = {
+  buttonLoader: "calcite-button--loader",
+  content: "content",
+  icon: "icon",
+  iconStart: "icon--start",
```

Comment:
I don't think we need BEM here at all

Reply:
Basing this on what we're doing in PickListItem for `actions--end` and `actions--start`.